docs/pages/components/form.md

```diff
@@ -81,7 +81,7 @@ Do not use the input field if:
     <br />
     <div class="fd-form-item">
         <label class="fd-form-label" for="input-05">Disabled Input:</label>
-        <input class="fd-input is-disabled" type="text" id="input-05" placeholder="Field placeholder text">
+        <input class="fd-input is-disabled" type="text" id="input-05" disabled placeholder="Field placeholder text">
```

Comment:
you can remove `is-disabled` class. The idea was that the user could use either `is-disabled` class or `disabled` attr interchangeably

Reply:
Ok, `is-disabled` class removed.

I don't think there is a point of creating `is-disabled` css class to actually disable a button, link or input. CSS wont cover things like _Tab_ key navigation or automatic focus on label click.